### ISSUE TITLE
DictionaryServer throws MatchError

### ISSUE THREAD

**Issue by chriscrabtree** [Baradoy/credo_spellchecker#1]
First, this is a great idea for a Credo Plugin! I was happy to find it--thank you for creating it. :-)

Of course, the ecosystem has changed a bit since `credo_spellchecker` was released. So it's entirely natural there would be some small issues from time to time.

I've already repaired this issue in my deps copy of the plugin. If you're open to a PR on this, I'm happy to offer it.

## Details
Running `mix credo` with these relevant dependencies:

- `{:credo, "~> 1.7", only: [:dev, :test], runtime: false}`
- `{:credo_spellchecker, "~> 0.1.4", only: [:dev, :test], runtime: false}`
- erlang 26.2.1
- elixir 1.16.0-otp-26

Leads to this output:
```
Checking 10 source files ...
Error while running Elixir.CredoSpellchecker.NoMisspelledWords on lib/my_project/repo.ex

07:31:15.042 [error] Task #PID<0.1823.0> started from #PID<0.1797.0> terminating
** (MatchError) no match of right hand side value: {:error, {:already_started, #PID<0.1861.0>}}
    (credo_spellchecker 0.1.4) lib/credo_spellchecker/dictionary_server.ex:14: CredoSpellchecker.DictionaryServer.get_dictionaries/1
    (credo_spellchecker 0.1.4) lib/credo_spellchecker/no_misspelled_words.ex:35: CredoSpellchecker.NoMisspelledWords.run/2
    (credo_spellchecker 0.1.4) lib/credo_spellchecker/no_misspelled_words.ex:2: CredoSpellchecker.NoMisspelledWords.do_run_on_source_file/3
    (elixir 1.16.0) lib/task/supervised.ex:101: Task.Supervised.invoke_mfa/2
    (elixir 1.16.0) lib/task/supervised.ex:36: Task.Supervised.reply/4
Function: &:erlang.apply/2
    Args: [#Function<0.67292521/1 in CredoSpellchecker.NoMisspelledWords.do_run_on_all_source_files/3>, [%SourceFile<lib/codetrance82/repo.ex>]]
** (EXIT from #PID<0.98.0>) an exception was raised:
    ** (MatchError) no match of right hand side value: {:error, {:already_started, #PID<0.1861.0>}}
        (credo_spellchecker 0.1.4) lib/credo_spellchecker/dictionary_server.ex:14: CredoSpellchecker.DictionaryServer.get_dictionaries/1
        (credo_spellchecker 0.1.4) lib/credo_spellchecker/no_misspelled_words.ex:35: CredoSpellchecker.NoMisspelledWords.run/2
        (credo_spellchecker 0.1.4) lib/credo_spellchecker/no_misspelled_words.ex:2: CredoSpellchecker.NoMisspelledWords.do_run_on_source_file/3
        (elixir 1.16.0) lib/task/supervised.ex:101: Task.Supervised.invoke_mfa/2
        (elixir 1.16.0) lib/task/supervised.ex:36: Task.Supervised.reply/4

```

**Comment by Baradoy** [comment]
> I've already repaired this issue in my deps copy of the plugin. If you're open to a PR on this, I'm happy to offer it.

Yes, @chriscrabtree I would love to see a PR for that, thanks!

**Comment by chriscrabtree** [comment]
[https://github.com/Baradoy/credo_spellchecker/pull/2](url)

**Comment by Baradoy** [comment]
Thanks for the PR, the fix is published in v0.1.5
